src/main/java/com/dev/cinema/service/MovieSessionService.java

```diff
@@ -7,5 +7,7 @@
 public interface MovieSessionService {
     List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
 
+    MovieSession get(Long id);
```

Comment:
it is not necessary to add methods to the service layer that you do not use from the controller layer

Reply:
Ok. np.